src/_shared/js/creatives.js

```diff
@@ -1,8 +1,9 @@
 import { write } from './dom.js';
 
-export function cleanupButtons(moreTexts) {
+export function cleanupButtons() {
+    const moreTexts = ['[%Offer1LinkText%]', '[%Offer2LinkText%]', '[%Offer3LinkText%]', '[%Offer4LinkText%]'];
```

Comment:
I'm not against this change, just be aware that if one day we have a creative with more or less than 4 items, you will need to revert it 😅